src/stories/select.stories.mdx

```diff
@@ -25,6 +25,14 @@ import gpioOptions from './assets/gpio-options'
       description: 'The placeholder value',
       table: { defaultValue: { summary: '' } }
     },
+    showpill: { 
+      description: 'when in multi-select variant, have the option to show pills in the select representing selected values',
+      table: { defaultValue: { summary: 'false' } }
```

Comment:
nit: If you want to make this controllable you can add `control: { type: 'boolean' }` here. You'd also have to add something like this to each Story too

<img width="753" alt="image" src="https://user-images.githubusercontent.com/16711130/204652700-b6931f65-bbf6-4157-bfa8-6b85504b287a.png">

Can reference the `match` and `open` properties in `dropdown.strories.mdx` if you want to make this change, but
it's not super necessary - I think having the example with pills you have already is probably enoug